src/Mesh/babylon.geometry.ts

```diff
@@ -635,88 +635,101 @@
             return serializationObject;
         }
 
+        private toNumberArray(origin: Float32Array | IndicesArray) : number[] {
+            if (Array.isArray(origin)) {
+                return origin;
+            } else {
+                var len = origin.length;

```

Comment:
this could be done like this: http://jsfiddle.net/wTJDy/ 
var aa =  Array.prototype.slice.call(a);